test/WebJobs.Script.Tests/Configuration/HostJsonFileConfigurationSourceTests.cs

```diff
@@ -97,6 +97,23 @@ public void MissingVersion_ThrowsException()
             Assert.StartsWith("The host.json file is missing the required 'version' property.", ex.Message);
         }
 
+        [Fact]
+        public void Version3_ThrowsException()
+        {
+            string hostJsonContent = @"
+            {
+              'version': '3.0',
+              'functions': [ 'FunctionA', 'FunctionB' ]
+            }";
+            TestMetricsLogger testMetricsLogger = new TestMetricsLogger();
+
+            File.WriteAllText(_hostJsonFile, hostJsonContent);
+            Assert.True(File.Exists(_hostJsonFile));
+
+            var ex = Assert.Throws<HostConfigurationException>(() => BuildHostJsonConfiguration(testMetricsLogger));
+            Assert.StartsWith("Invalid setting for host.json 'version' property of '3.0'", ex.Message);
```

Comment:
- This statement is a little confusing `The value for 'version' property of {ScriptConstants.HostMetadataFileName} is invalid. `
- Also `We recommend you set the version to '2.0'.`